File: engine/model/src/main/scala/tools/packet/quest/info/UpdateQuestInfo.scala
package tools.packet.quest.info

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class UpdateQuestInfo(private var _questId: Short, private var _npcId: Int) extends PacketInput {
  def questId: Short = _questId

  def npcId: Int = _npcId

  override def opcode(): SendOpcode = SendOpcode.UPDATE_QUEST_INFO
}

File: engine/engine-model/src/main/java/tools/packet/quest/UpdateQuestInfo.java
package tools.packet.quest;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record UpdateQuestInfo(Short questId, Integer npcId) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.UPDATE_QUEST_INFO;
   }
}